src/calculateRentalCost.js

```diff
@@ -5,6 +5,25 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const shortTime = 2;
```

Comment:
The variable 'shortTime' is not used anywhere in your code. Such unused variables should be removed to keep the code clean and understandable.